treeherder/webapp/api/performance_data.py

```diff
@@ -171,6 +171,10 @@ def list(self, request, project):
                                              'signatures, push_id, or '
                                              'job_id')
 
+        if not all([type(id) == int for id in job_ids]):
+            return Response({"message": "Job id(s) ,must be specified as integers"},
```

Comment:
there's a stray comma before "must" here. I think I can just fix that before merging though.